site/.vitepress/config.ts

```diff
@@ -45,7 +45,7 @@ export default defineConfig({
         items: [
```

Comment:
In general always good to break up these "new" things into a seperate PR. that way we can rollback faster, or unblock the light account docs faster. But thanks for doing this!!